### DISCUSSION TITLE
flatbuffer support Arrays in mcap

### DISCUSSION THREAD

**Discussion by ADaiZ** [foxglove/mcap#1075] [General]
we make mcap file by flatbuffer , then studio throw new Error("Unions and Arrays are not supported in mcap-support currently.")
.fbs file like this
struct SwitchStatus{
  data:[ubyte:16];
}

**Comment by ADaiZ** [comment]
i don't know how to fix this
can you help me? please

**Reply by defunctzombie** [reply]
Please provide your mcap file that is not working.
